mw-webapp/src/component/table/Table.tsx

```diff
@@ -1,40 +1,28 @@
-import {useEffect, useState} from "react";
-import {
-  flexRender,
-  getCoreRowModel,
-  useReactTable,
-} from "@tanstack/react-table";
-import {columns} from "src/component/table/columns";
-import {getDayReports} from "src/dataAccessLogic/getDayReports";
-import {DayReport} from "src/model/businessModel/DayReport";
+import {memo} from "react";
+import {flexRender} from "@tanstack/react-table";
+import {TableReports} from "src/pages/reportsTable/ReportsTable";
```

Comment:
unconsistency TableReports in ReportsTable

Reply:
done